cmd/skupper/skupper.go

```diff
@@ -820,6 +823,11 @@ the .bash_profile. i.e.: $ source <(skupper completion)
 type cobraFunc func(cmd *cobra.Command, args []string)
 
 func newClient(cmd *cobra.Command, args []string) {
+	if cmd.Name() == "version" {
```

Comment:
Not an out and out objection, but this feels a little ugly to me.

What about having an alternative to newClient, that does not exit, but records any error (and leaves the cli var unset? This could then be used as the parameter to NewCmdVersion, whose RunE function could then check whether the cli had been set before invoking on it and print the error if not?

Reply:
I initially started down that path but did not work it all the way thru, will make another pass at it as it would be cleaner for sure...